pcl_ros/src/pcl_ros/filters/filter.cpp

```diff
@@ -151,7 +151,7 @@ pcl_ros::Filter::subscribe()
     // Subscribe in an old fashion to input only (no filters)
     sub_input_ =
       this->create_subscription<PointCloud2>(
-      "input", max_queue_size_,
+      "input", rclcpp::SensorDataQoS().keep_last(max_queue_size_),
```

Comment:
Good catch. Can you update the other version to use this same `rclcpp::SensorDataQoS()` format for consistency?

Reply:
I updated QoS format in `use_indices` section.